org.eclipse.scout.rt.ui.html/src/main/java/org/eclipse/scout/rt/ui/html/json/JsonRequestHelper.java

```diff
@@ -118,6 +118,17 @@ public JSONObject createVersionMismatchResponse() {
     return response.toJson();
   }
 
+  /**
+   * UI Text <code>NetSystemsNotAvailable</code>
```

Comment:
This is unnecessary. I would remove it from createUnsafeUploadResponse and createRejectedUploadResponse as well.